### ISSUE TITLE
SwiftSoup.parse(_:_:) fails to correctly parse this iframe

### ISSUE THREAD

**Issue by NSExceptional** [scinfu/SwiftSoup#18]
<iframe src=\"https://www.googletagmanager.com/ns.html?id=GTM-M48W9J\"\n\t\t                  height=\"0\" width=\"0\" style=\"display:none;visibility:hidden\"></iframe>

Here are its attributes:

```swift
[
    "src": SwiftSoup.Attribute,
    "": SwiftSoup.BooleanAttribute,
    "height": SwiftSoup.Attribute,
    "width": SwiftSoup.Attribute,
    "style": SwiftSoup.Attribute
 ]
```

It crashes in `ParseSettings.normalizeAttributes()` on this line:

```swift
try attr.setKey(key: attr.getKey().lowercased())
```

because `attr.getKey()` is empty.

## Code to reproduce:

```swift
SwiftSoup.parse(html, "https://9to5mac.com")
```

**Comment by scinfu** [comment]
Can you give me the html you are using ?

**Comment by scinfu** [comment]
Parsing your iframe i found a bug and i fixed it with in version 1.3.1.
Can you provide a test html where it crash ?
Thank you

**Comment by NSExceptional** [comment]
```swift
SwiftSoup.parse(html, "https://9to5mac.com")
```

and

```swift
SwiftSoup.parse(html, "http://www.vox.com")
```

**Comment by scinfu** [comment]
temporarly you can fix this error commenting `Validate.swift` line 103 
// throw Exception.Error(type: ExceptionType.IllegalArgumentException, Message: "String must not be empty")

in the html maybe there is and attribute with empty key or the library parse the html and insert an empty attribute key, when the library validate the key it not pass: 
`"": SwiftSoup.BooleanAttribute`
I'm trying to fix it .

**Comment by scinfu** [comment]
or you can encode the string in utf16 instead utf8:

`var html = try String(contentsOf: myURL, encoding: .utf16)`

**Comment by NSExceptional** [comment]
Thanks!

**Comment by scinfu** [comment]
sorry maybe parsing in utf16 is not a good idea . comment line 103 while i'm fix this issue :)

**Comment by scinfu** [comment]
Ok now its fixed and you can update to version 1.3.2 , 
[1.3.2](ad3c19d078fba17fba046f8434b3a1fad8ac8a3c).
I'm closing this issue. Please feel free to reopen it if needed
